tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.cpp

```diff
@@ -19,59 +19,70 @@ TEST_P(Newton3OnOffTest, countFunctorCallsTest) {
-            if (containerOption == autopas::ContainerOption::verletLists or
-                containerOption == autopas::ContainerOption::verletListsCells or
-                containerOption == autopas::ContainerOption::verletClusterLists or
-                containerOption == autopas::ContainerOption::varVerletListsAsBuild or
-                containerOption == autopas::ContainerOption::verletClusterCells) {
-              continue;
-            }
-
-            containerSelector.selectContainer(containerOption, containerInfo);
-
-            auto container = containerSelector.getCurrentContainer();
-
-            for (auto traversalOption : container->getAllTraversals()) {
-              if (traversalOption == autopas::TraversalOption::lc_c01 or
-                  traversalOption ==
-                      autopas::TraversalOption::lc_c01_combined_SoA /*and autopas::autopas_get_max_threads() > 1*/) {
-                continue;
+    Combine(ValuesIn([]() -> std::vector<std::tuple<autopas::ContainerOption, autopas::TraversalOption>> {
+              // needed because CellBlock3D (called when building containers) logs always
+              autopas::Logger::create();
+
+              std::vector<std::tuple<autopas::ContainerOption, autopas::TraversalOption>> ret;
+
+              // container factory
+              autopas::ContainerSelector<Particle> containerSelector({0., 0., 0.}, {10., 10., 10.}, 1.);
+              autopas::ContainerSelectorInfo containerInfo(1., 0., 64, autopas::LoadEstimatorOption::none);
+
+              // generate for all containers, even those to come
+              for (auto containerOption : autopas::ContainerOption::getAllOptions()) {
+                // skip containers that do not work with both newton modes
+                // @TODO: let verlet lists support Newton 3
+                if (containerOption == autopas::ContainerOption::verletLists or
+                    containerOption == autopas::ContainerOption::varVerletListsAsBuild or
+                    containerOption == autopas::ContainerOption::verletClusterCells) {
+                  continue;
+                }
+
+                containerSelector.selectContainer(containerOption, containerInfo);
+
+                auto container = containerSelector.getCurrentContainer();
+
+                for (auto traversalOption : container->getAllTraversals()) {
+                  // exclude traversals that do not support Newton3
+                  if (traversalOption == autopas::TraversalOption::lc_c01 or
+                      traversalOption ==
+                          autopas::TraversalOption::lc_c01_combined_SoA /*and autopas::autopas_get_max_threads() > 1*/
+                      or traversalOption == autopas::TraversalOption::vlc_c01 or
+                      traversalOption == autopas::TraversalOption::vcl_cluster_iteration or
+                      traversalOption == autopas::TraversalOption::vcl_c01_balanced) {
```

Comment:
Didn't you say that you wanted to use `isApplicable` here?

Reply:
This is what happens when you review an unfinished draft you were not asked to review.

Reply:
I unticked the tick of the change :P